src/main/groovy/com/calgaryscientific/gradle/VeracodeUploadFileSandboxTask.groovy

```diff
@@ -63,13 +63,11 @@ class VeracodeUploadFileSandboxTask extends VeracodeTask {
     void run() {
         Integer maxTries = Integer.parseInt((this.maxUploadAttempts != null) ? this.maxUploadAttempts : '10')
         Integer waitTime = Integer.parseInt((this.waitTimeBetweenAttempts != null) ? this.waitTimeBetweenAttempts : '5000')
-        getFileSet().each { file ->
-            VeracodeUploadFile.uploadFile(file, maxTries, waitTime, veracodeAPI, getOutputFile(), true)
-            if (delete == "true") {
-                println("Deleting ${file}")
-                file.delete()
-            }
+        Boolean deleteBool = false
```

Comment:
I think you can just use .toBoolean() per this: http://mrhaki.blogspot.ca/2009/11/groovy-goodness-convert-string-to.html